src/app/views/experiment_page.js

```diff
@@ -0,0 +1,68 @@
+import View from 'ampersand-view';
+import dom from 'ampersand-dom';
+import mustache from 'mustache';
+import webChannel from '../lib/web_channel';
+
+export default View.extend({
+  template: mustache.render(`<section class="page" data-hook="experiment-page">
+               <h1 data-hook="displayName"></h1>
+               <button data-hook="install">Install</button>
+             </section>`),
+
+  events: {
+    'click button': 'onClick'
+  },
+
+  initialize(opts) {
+    View.prototype.initialize.apply(this, arguments);
```

Comment:
No need to do this for `initialize`: https://github.com/AmpersandJS/ampersand-view/blob/master/ampersand-view.js#L152